src/Misc.hs

```diff
@@ -88,7 +88,13 @@ numToHex num = btsToHex (unpack (toLazyByteString (word64BE (doubleToWord num)))
 -- "68-65-6C-6C-6F"
 -- >>> strToHex "world"
 -- "77-6F-72-6C-64"
+-- >>> strToHex ""
+-- "--"
+-- >>> strToHex "h"
+-- "68-"
```

Comment:
@yegor256 these function call results are auto calculated in Haskell. So we can claim them as micro tests. Should be enough for such small bug